src/compas/geometry/primitives/frame.py

```diff
@@ -175,9 +175,9 @@ def yaxis(self):
     def yaxis(self, vector):
         yaxis = Vector(*vector)
         yaxis.unitize()
-        zaxis = Vector.cross(self.xaxis, yaxis)
+        zaxis = Vector(*cross_vectors(self.xaxis, yaxis))
         zaxis.unitize()
-        self._yaxis = Vector.cross(zaxis, self.xaxis)
+        self._yaxis = Vector(*cross_vectors(zaxis, self.xaxis))
```

Comment:
`zaxis` is a `Vector`.
this could be left as-is or changed to `zaxis.cross(self.xaxis)`